superset/datasets/api.py

```diff
@@ -181,6 +181,7 @@ class DatasetRestApi(BaseSupersetModelRestApi):
     @protect()
     @safe
     @statsd_metrics
+    @event_logger.log_this
```

Comment:
It's not good to have duplicate statsd metrics being sent, we can change the `log_this` decorator signature so we can pass a parameter to it (does it imply a bit refactor) or address this later

Reply:
Done!